toolbox/incari/object/get-position.md

```diff
@@ -0,0 +1,31 @@
+# Overview
+
+![The Get Position Node.](../../../.gitbook/assets/toolbox/incari/object/get-position.PNG)
+
+| Input | Output |
+| :--- | :--- |
+| Object ID | Position |
+
+**Get Position** returns the *position* of an **Object** in a **Scene**. The **Object ID** is given as input to the **Node**.
+
```

Comment:
# Attributes

|Attribute|Type|Description|
|---|---|---|
|`Object`|**ObjectID**|The **Object** that you wish to return the `position` of, if one isn't provided in the  `Object ID` **Socket**.|